### DISCUSSION TITLE
ModeSBeast as receiver

### DISCUSSION THREAD

**Discussion by nowiresfil** [ketilmo/balena-ads-b#93] [Q&A] [answered]
Gday,
Is it possible to configure Balena-adsb to use the ModeSBeast USB receiver as input, instead of the RTLSDR tuner?

**Comment by ketilmo** [comment]
Hi @nowiresfil,

Thanks for reaching out! Out of the box, only USB devices currently. I have created a feature request for this idea, which you can find here: #99

**Reply by ketilmo** [reply, marked as answer]
Thanks to @shawaj, this functionality is now in place! 👏🏻
